Changelog.md

```diff
@@ -1,3 +1,6 @@
+**V1.10.8 - Updates**
+- Fix Meade parsing from INDI clients.
+
 **V1.10.6 - Updates**
```

Comment:
It looks like you need to fetch and rebase your changes onto develop, you created your branch quite a while ago:
```bash
# Add an upstream URL
# This URL is for ssh, change it if you use HTTPS
git remote add upstream git@github.com:OpenAstroTech/OpenAstroTracker-Firmware.git
git fetch -p upstream
git rebase -i origin/develop # Interactive rebase
# Fix merge conflicts
git push -f # Force push is needed because the commits have changed
```